src/tlo/methods/enhanced_lifestyle.py

```diff
@@ -660,7 +660,7 @@ def update_all_properties(self, df):
         :param df: The population dataframe """
         # get months since last poll
         now = self.module.sim.date
-        months_since_last_poll = round((now - self.date_last_run) / np.timedelta64(1, "M"))
+        months_since_last_poll = round((now - self.date_last_run) / np.timedelta64(1, "m"))
```

Comment:
is that right?

Reply:
rolled back. Coding for month is "M", which is what we want and what was there originally. ("m" is the the code for minutes".

https://numpy.org/doc/stable/reference/arrays.datetime.html

Reply:
I don't know why but this causes errors on my machine! It won't run with "M"... I'll check out why this is happening when I'm back.